ports/openimageio/portfile.cmake

```
@@ -1,15 +1,22 @@
+vcpkg_download_distfile(FIX_LIBRAW_BUILD_PATCH
+    URLS https://github.com/AcademySoftwareFoundation/OpenImageIO/commit/904df59ab74d0c89c1c9eea7d5ef0ecfe0620b2c.diff?full_index=1
+    FILENAME Fix-libraw-build.patch
```

Nitpick:
```suggestion
    FILENAME AcademySoftwareFoundation-OpenImageIO-libraw-build.patch
```
to group files in `downloads/`.

---

Is there any reason why there isn’t a vcpkg_patch_from_github function that codifies this pattern @dg0yt ? Was thinking about adding it.

---

Only few people care about names in `downloads/`.

TBH I would probably want to be able to directly add 
~~~
  https://this.patch:SHA512=abc
~~~
or similar to the `PATCHES` list.